pkg/apis/kubeone/types.go

```diff
@@ -363,6 +363,9 @@ type Features struct {
 	MetricsServer *MetricsServer `json:"metricsServer,omitempty"`
 	// OpenIDConnect
 	OpenIDConnect *OpenIDConnect `json:"openidConnect,omitempty"`
+	// Encryption Providers
+	// +k8s:conversion-gen=false
```

Comment:
This is going to disable conversion-gen for v1beta1 as well, and we need the conversion function there.

We should remove this comment and instead create a custom Convert function for v1alpha1 that would do nothing for EncryptionProviders.

Reply:
Yeah, there are examples of this in code already, with any missing fields from v1apha1